recordings/trainingAndRecognition/manuelTraining.py

```diff
@@ -0,0 +1,6 @@
+from piwho import recognition
+recog = recognition.SpeakerRecognizer()
+# set the name of the speaker
+recog.speaker_name = 'Ruth1'
+# create dataset of the files in the directory
+recog.train_new_data('/home/pi/jasper/recordings/')
```

Comment:
Da der Ordnerpfad doch öfters benutzt wird, kann der als globale Variable in die `__init__.py` des `recordings`' package.